packages/f-header/test-utils/component-objects/f-header.component.js

```diff
@@ -1,5 +1,6 @@
 const loginLink = () => $('[data-test-id="login-link"]');
-const offersLink = () => $('[data-test-id="offers-link]"');
+const offersLinkMobile = () => $('[data-test-id="offers-link-mobile]"');
+const offersLinkDesktop = () => $('[data-test-id="offers-link-desktop]"');
```

Comment:
Is there a way we can consolidate this into a single selector that can be used for both mobile and desktop?

Reply:
Could we have two data-test-ids on each link, one "offers-link-mobile/desktop" for the mobile/desktop tests and one "offers-link" for generic use?

Reply:
Wouldn't the attribute starts-with selector `$('[data-test-id^="offers-link"]')` suffice for [the case where we need to match either]? Also the closing quotes and square-parens are in the wrong order above: 
```suggestion
const offersLinkDesktop = () => $('[data-test-id="offers-link-desktop"]');
```